modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-upgrading-lvms-on-sno_{context}"]
+= Upgrading {lvms} on {sno} clusters
+
+Currently, it is not possible to upgrade from {rh-storage} Logical Volume Manager Operator 4.11 to {lvms} 4.12 on {sno} clusters.
+
+.Procedure
```

Comment:
![image](https://user-images.githubusercontent.com/77501157/219758938-ac0f38ab-825a-4c58-baf7-49f2f40c05a2.png)
Seems like the .Procedure heading is getting pulled into this note, which I don't think is what you want. The admonition itself is fine, but I think it may be the placement directly after .Procedure that's bugging this out. Perhaps move the note to before. Maybe more of an IMPORTANT just because data loss is a biggie.